train.py

```diff
@@ -101,11 +97,79 @@ def printTree(root):#this is for testing purposes
             print("Internal"+str(current.splitAttribute)+" "+str(current.threshold))
             open.append(current.leftChild)
             open.append(current.rightChild)
-        
-    
+            
+            
+def getBestInfoGain(data_cases,attrib):
     
+    if data_cases[0].attributesAlreadyExamined[attrib]:
+            
+        return -1, -1
```

Comment:
Might want a comment here, to explain why returning `-1, -1`, that it's to indicate that this attribute has already been examined for these data cases. Otherwise, it's kinda magic numbers